File: 01_Basico/01 - Conceitos_Basicos/Respostas_em_Python/questao02extra.py
numero_inteiro1 = int(input("Digite um número inteiro: "))
numero_inteiro2 = int(input("Digite um número inteiro: "))
print(f"O primeiro: {numero_inteiro1}\nO segundo é: {numero_inteiro2}")
print("Invertendo...")

numero_inteiro1, numero_inteiro2 = numero_inteiro2, numero_inteiro1
print(f"O primeiro: {numero_inteiro1}\nO segundo é: {numero_inteiro2}")

File: 01_Basico/01 - Conceitos_Basicos/Respostas_em_Kotlin/questao02extra.kt
fun main() {
    var numero_inteiro1 = 5
    var numero_inteiro2 = 10
    println("O primeiro é : $numero_inteiro1\nO segundo é: $numero_inteiro2")
    println("Invertendo...")
    numero_inteiro1 = numero_inteiro2.also { numero_inteiro2 = numero_inteiro1 }
    println("O primeiro é: $numero_inteiro1\nO segundo é: $numero_inteiro2")
}